src/Integration.Vsix/SonarLintDaemon/SonarLintDaemonSplashscreen.xaml

```diff
@@ -1,16 +1,16 @@
 ﻿<Window x:Class="SonarLint.VisualStudio.Integration.Vsix.SonarLintDaemonSplashscreen"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        Title="Activate support for JavaScript in SonarLint" Width="500"
+        Title="Activate support for additional languages in SonarLint" Width="500"
         WindowStartupLocation="CenterScreen" Topmost="True" SizeToContent="Height"
         >
     <StackPanel Margin="5">
         <TextBlock Margin="0,5"
-            Text="SonarLint for Visual Studio can also analyze JavaScript files."
+            Text="SonarLint for Visual Studio can also analyze JavaScript, C or C++ files."
```

Comment:
Not sure @christophelevis receive GitHub notifications. I asked him directly and he said this is fine like this.